template.tpl

```diff
@@ -1,11 +1,3 @@
-﻿﻿___TERMS_OF_SERVICE___
```

Comment:
Need to leave this in, otherwise it might not get picked up to update in the gallery